src/assets/colors.scss

```
@@ -7,6 +7,7 @@ $color-white: #fefefe;
 $color-gray: #60646d;
 $gray-70: #666666;
 $gray-60: #808080;
+$gray-30: #d3d3d3;
```

I recommend organizing variables into color-coded groups to enhance file organization and maintainability. While not within the scope of this issue, it's a change we can gradually integrate over time. What do you think? Letmme know plz

```suggestion
// White
...

// Gray
$gray-20: #dfe1e1;
$gray-30: #d3d3d3;
$gray-60: #808080;
$gray-70: #666666;

// Blue
$blue-20: #e4faff; 

// Rest
...
```